core/client/fs/src/main/java/alluxio/client/block/AlluxioBlockStore.java

```diff
@@ -75,19 +75,18 @@ public static AlluxioBlockStore create() {
    * @return the {@link AlluxioBlockStore} created
    */
   public static AlluxioBlockStore create(FileSystemContext context) {
-    return new AlluxioBlockStore(context, NetworkAddressUtils.getClientHostName());
+    return new AlluxioBlockStore(context, TieredIdentityFactory.getInstance());
```

Comment:
`TieredIdentityFactory.getInstance` returns the `TieredIdentity` for the current host, should we rename the method to make it more clear?

Reply:
renamed to `localIdentity`